Cesium.IntegrationTests/pointers/pointers_substraction.ignore.c

```diff
@@ -0,0 +1,7 @@
+int main(int argc, char* argv[])
+{
+    int a[10];
+    int* b = &a[1];
+    int* c = &a[8];
+    return 6 * (c - b);
+}
```

Comment:
Please rename this file to `pointers_subtraction.ignore.c`.

Reply:
Eventually I fix my bad habits